stash/src/controller/mgxAirdropController.ts

```diff
@@ -0,0 +1,173 @@
+import { Request, Response } from 'express'
+import {
+  getEligibilityAtBlockN,
+  verifySignature,
+} from '../service/MgxAirdropService.js'
+import { mgxAirdropRepository } from '../repository/MgxAirdropRepository.js'
+import * as errorHandler from '../error/Handler.js'
+import { BN_ZERO } from 'gasp-sdk'
+import { BadRequestException } from '../error/Exception.js'
+import {
+  mgxAirdropEligibilitySchema,
+  mgxAirdropLinkAddressSchema,
+  mgxAirdropSignatureSchema,
+} from '../schema/MgxAirdropSchema.js'
+
+const FIRST_SNAPSHOT_BLOCK_HASH =
+  '0x8de8328944b57a0fae7b6780d98c8d98e31a8539a393b64e299a38db0f200edf'
+const SECOND_SNAPSHOT_BLOCK_HASH =
+  '0x90f9f232b2a8e142a5b268effa53aa47963739dfa73a893962a42f67be60e282'
+
+const SNAPSHOTS = [FIRST_SNAPSHOT_BLOCK_HASH, SECOND_SNAPSHOT_BLOCK_HASH]
+
+export async function checkEligibility(req: Request, res: Response) {
+  /*
+    #swagger.tags = ['mgx-airdrop']
+    #swagger.summary = 'Check eligibility for MGX airdrop'
+    #swagger.description = 'Verifies if the provided address is eligible for the airdrop based on snapshots.'
+    #swagger.responses[200] = {
+      description: 'Successful response',
```

Comment:
Add here in path parameter you can check the tracing how its done similar to    #swagger.parameters['txHashOrEntityId'] = {
     in: 'path',